app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -257,7 +257,6 @@ public void onValueChanged(@NonNull Boolean loading) {
                         // We start a transition only if a page was just loading before
                         // allowing to avoid issue #1179
                         backgroundTransitionGroup.startTransition(ANIMATION_DURATION);
-                        progressView.setProgress(progressView.getMax());
```

Comment:
This was the simplest fix for the issue while also not regressing #1179. Ultimately I would love to revisit with #2394.

Will this potentially break something else? I ran tested it a couple times and nothing jumped out at me.

Reply:
Mhhhh. I think we added that because otherwise the "finished loading" animation was missing. Can you still see it? If it's still there and "git blame" for this line doesn't reveal any other reason then let's land this. :)

In any other case I think it would be better to regress #1179 (~ polish) than to keep #1837 (broken). :)